### PR TITLE
Clean up ns and make code more idiomatic

### PR DESCRIPTION
**Pull request by danielcompton**
Some of these changes you may not prefer, I'm happy to pull them back out if you'd like.

### CODE REVIEW

File: src/loom/alg.clj [hunk 1]
```diff
@@ -432,9 +432,9 @@ can use these functions."
                       color (- 1 (coloring v))
                       nbrs (graph/successors g v)]
                   ;; TODO: could be better
-                  (if (some #(and (coloring %) (= (coloring v) (coloring %)))
+                  (when-not (some #(and (coloring %) (= (coloring v) (coloring %)))
                             nbrs)
-                    nil ;not bipartite
+                    ;not bipartite
```
Comment (danielcompton) [inline src/loom/alg.clj]: I didn't quite understand the purpose of this comment.

**Comment by danielcompton** [comment]
I've signed the Clojure CA and am happy to sign over copyright. I'm a little confused by the idea though, I wouldn't expect signing the Clojure CA would apply to contributions to Loom when it isn't a Core Clojure project?

**Comment by aysylu** [comment]
There's no longer a requirement to sign the Clojure CA in order to contribute to Loom. I've announced it some time ago, but it looks like not all the references were updated. The [FAQ page](http://aysy.lu/loom/faq.html#features) should now have the up-to-date information. Sorry for the confusion. The vision behind Loom is to eventually make it a Clojure contrib library, which is why the requirement existed before.

Thank you for your great contributions! I'm still in the process of reviewing the 2 PRs and will get back to you shortly.

File: src/loom/alg.clj (hunk 1 above)
Comment (aysylu) [inline src/loom/alg.clj]: In this case, we should leave the if-statement to make it clear that the true branch of the if-statement, which returns nil, indicates that the graph is not bipartite.

**Comment by aysylu** [comment]
Everything looks good. If you could revert just that one change that we discussed in src/loom/alg.clj on line 437, I'll merge the PR.

**Comment by danielcompton** [comment]
All fixed.

**Comment by aysylu** [comment]
Great, thanks!

**Pull request merged**